src/midgard/tiles.cc

```diff
@@ -81,7 +88,11 @@ template <class coord_t> struct closest_first_generator_t {
     auto y0 = tiles.TileBounds().miny() + y * tiles.SubdivisionSize();
     auto y1 = tiles.TileBounds().miny() + (y + 1) * tiles.SubdivisionSize();
     auto distance = std::numeric_limits<double>::max();
-    std::list<coord_t> corners{{x0, y0}, {x1, y0}, {x0, y1}, {x1, y1}};
+    corners.clear();
```

Comment:
Another option would be to remove the `corners` container altogether.  Just have one `coord_t corner` and unroll the loop, checking for a smaller distance on a "as you go" basis.

Reply:
In the spirit of "maximum performance".  :)

Reply:
I thought about trying that - I'll test locally and see if it makes any further improvement.

Reply:
No noticeable gain by unrolling the loop.

Reply:
👍